src/megatron/bridge/models/mimo/llava_provider.py

```
@@ -0,0 +1,137 @@
+    vocab_size: int = 32256  # Vicuna vocab size
+    
+    # Optional custom configs
+    language_config: Optional[TransformerConfig] = None
+    
+    def __post_init__(self):
+        """Build specs after initialization."""
+        if self.vision_encoder_module is None:
+            raise ValueError(
+                "vision_encoder_module must be provided. "
+                "Example: LlavaMimoProvider(vision_encoder_module=HFCLIPEncoder, ...)"
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)